### PR TITLE
Lp new login

### PR DESCRIPTION
**Pull request by lspdrz**
https://user-images.githubusercontent.com/9705113/145461960-670b7791-557a-4424-a2b7-e900e3adbec7.mov

[sc-1384]

DO NOT MERGE UNTIL https://github.com/Ameelio/connect-api-nest/pull/559 is merged and deployed

Changes the login page to include state selection

### CODE REVIEW

File: src/utils/enums/SupportedUSState.enum.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export enum SupportedUSState {
+    'CO' = 'Colorado', // should I add later?
```
Comment (lspdrz) [inline src/utils/enums/SupportedUSState.enum.ts:2]: @gabrielsaruhashi should we add Colorado to the dropdown now or later?

File: src/pages/Login/index.tsx [hunk 2]
```diff
@@ -81,15 +84,25 @@ const LoginContainer: React.FC = () => {
...
   };
 
-  const facilityOptions = facilities.map((facility: Facility) => (
-    <Option key={`option-${facility.id}`} value={facility.id}>
-      {facility.name}
-    </Option>
-  ));
+  const stateOptions = Object.keys(SupportedUSState).map((supportedState) => (
+        <Option key={`option-${supportedState}`} value={supportedState}>
+          {SupportedUSState[supportedState as keyof typeof SupportedUSState]}
```
Comment (cainlevy) [inline src/pages/Login/index.tsx:89]: might be able to bypass this type mismatch with `Object.entries`

File: src/pages/Login/index.tsx [hunk 3]
```diff
@@ -145,16 +160,31 @@ const LoginContainer: React.FC = () => {
...
+              label={t("placeholder.state")}
+              labelAlign={"left"}
+              rules={[{ required: true, message: "Must select a state." }]}
+            >
+              <Select onSelect={(value: UsState) => setState(value)}>{stateOptions}</Select>
+            </Form.Item>
+
+
+{
+  state && (
```
Comment (cainlevy) [inline src/pages/Login/index.tsx:174]: is prettier working?

**Review by wittejm — APPROVED** [review]

File: src/utils/enums/SupportedUSState.enum.ts (hunk 1 above)
Comment (cainlevy) [reply src/utils/enums/SupportedUSState.enum.ts:2]: can we get the list of states for the dropdown from the fetched facility data? that would prevent empty facility dropdowns, too.

we'd still need state names, presumably, but that could come from i18n or some other static file.

File: src/pages/Login/index.tsx (hunk 2 above)
Comment (lspdrz) [reply src/pages/Login/index.tsx:89]: Yep good idea. Or object.values works too turns out

File: src/pages/Login/index.tsx (hunk 3 above)
Comment (lspdrz) [reply src/pages/Login/index.tsx:174]: now it is!!

File: src/utils/enums/SupportedUSState.enum.ts (hunk 1 above)
Comment (lspdrz) [reply src/utils/enums/SupportedUSState.enum.ts]: Changed!!1 And yeah we still need the state names, I just used the enum for now

**Pull request merged**